packages/wonder-blocks-modal/components/modal-header.js

```diff
@@ -32,6 +32,11 @@ type Props = {|
      * use when the item is used on a dark background.
      */
     light: boolean,
+
+    /**
+     * An optional id to provide a selector for the title element.
+     */
+    id?: string,
```

Comment:
Maybe change this to `titleId?: string`.  Should this be mandatory?

Reply:
Agreed, this should be mandatory if we want to enforce accessibility.